test/unit/test_utils.py

```diff
@@ -42,6 +50,24 @@ def test_prep_stream_data_wrap(source):
     assert_prepped_stream_identity(test, io.BytesIO)
 
 
+class NullRawRSAKeyring(RawRSAKeyring):
```

Comment:
Two thoughts:
1. `IdentityKeyring` would probably be a better name for this: more representative of what it does.
2. Is there any reason you are subclassing from `RawRSAKeyring` rather than `Keyring`?

Reply:
The null keyrings were added when I was writing unit tests. I didn't use them later. I will delete them.